gateway-service/src/main/java/org/zowe/apiml/gateway/conformance/ParserFactory.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+
+package org.zowe.apiml.gateway.conformance;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.zowe.apiml.product.gateway.GatewayConfigProperties;
+
+import java.util.Map;
+
+
+public class ParserFactory {
```

Comment:
You can use lombok's `@UtilityClass`, method can be just public and no need for the private constructor